server_admin/topics/sso-protocols/oidc.adoc

```diff
@@ -97,6 +97,80 @@ This is used by clients running on internet-connected devices that have limited
 
+===== Client Initiated Backchannel Authentication Grant
+
+:tech_feature_name: Client Initiated Backchannel Authentication Grant
+:tech_feature_setting: -Dkeycloak.profile.feature.ciba=enabled
+include::../templates/techpreview.adoc[]
+
+This is used by clients who want to initiate the authentication flow by communicating with the OpenID Provider directly without redirect through the user's browser like OAuth 2.0's authorization code grant. Here's a brief summary of the protocol:
+
+. The client requests {project_name} an auth_req_id that identifies the authentication request made by the client. {project_name} creates the auth_req_id.
+. After receiving this auth_req_id, this client repeatedly needs to poll {project_name} to obtain an Access Token, Refresh Token and ID Token from {project_name} in return for the auth_req_id until the user is authenticated. 
+
+An administrator can configure Client Initiated Backchannel Authentication (CIBA) related operations as `CIBA Policy` per realm.
+
+====== CIBA Policy
+
+An administrator carries out the following operations on the `Admin Console` :
+
+- Open the `Authentication -> CIBA Policy` tab.
+- Configure items and click `Save`.
+
+The configurable items and their description follow.
+
+|===
+|Configuration|Description
+
+|Backchannel Token Delivery Mode
+|Specifying how the CD (Consumption Device) gets the authentication result and related tokens. There are three modes, "poll", "ping" and "push". {project_name} only supports "poll". The default setting is "poll". This configuration is required.
+ For more details, see https://openid.net/specs/openid-client-initiated-backchannel-authentication-core-1_0.html#rfc.section.5[CIBA Specification].
+
+|Expires In
+|The expiration time of the "auth_req_id" in seconds since the authentication request was received. The default setting is 120. This configuration is required.
+ For more details, see https://openid.net/specs/openid-client-initiated-backchannel-authentication-core-1_0.html#successful_authentication_request_acknowdlegment[CIBA Specification].
+
+|Interval
+|The interval in seconds the CD (Consumption Device) needs to wait for between polling requests to the token endpoint. The default setting is 5. This configuration is optional.
+ For more details, see https://openid.net/specs/openid-client-initiated-backchannel-authentication-core-1_0.html#successful_authentication_request_acknowdlegment[CIBA Specification].
+
+|Authentication Requested User Hint
+|The way of identifying the end-user for whom authentication is being requested. The default setting is "login_hint".  There are three modes, "login_hint", "login_hint_token" and "id_token_hint". {project_name} only supports "login_hint". The default setting is "poll". This configuration is required.
```

Comment:
The default setting is "login_hint" (instead of "poll") ?

Reply:
@vanrar68 Yes. That's typos. I will remove `The default setting is "poll"`. Thank you.